src/main/java/app/dao/impl/GameDAOImpl.java

```diff
@@ -96,14 +72,50 @@ public Game update(Long id, Game game) {
                 foundGame.setPlaytime(game.getPlaytime());
             if (game.getDescription() != null)
                 foundGame.setDescription(game.getDescription());
-            if (game.getPlatformSet() != null)
-                foundGame.setPlatformSet(game.getPlatformSet());
             if (game.getGenreSet() != null)
-                foundGame.setGenreSet(game.getGenreSet());
+                foundGame.setGenreSet(getFoundGenres(game));
+            if (game.getPlatformSet() != null)
```

Comment:
is there a reason for calling getFoundGenres(game) when setting to foundGame instead of game.getGenreSet when its already been set with game.setGenreSet(getFoundGenres(game)); further up?

Reply:
The idea was to match the naming convention of other similar circumstances where entities acquired by `em.find()` often get the prefix `found` like `foundGame` for instance. But it's not a necessary change